osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -3244,6 +3244,36 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/stjuno.svg"
       }
-    }
+    },
+	{
+      "description": "MUSE Governance Token",
+      "type_asset": "cw20",
+      "address": "juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3",
+      "denom_units": [
+        {
+          "denom": "IBC/6B982170CE024689E8DD0E7555B129B488005130D4EDA426733D552D10B36D8F",
+          "exponent": 0,
+          "aliases": ["cw20:juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3"]
+        },
+        {
+          "denom": "muse",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/6B982170CE024689E8DD0E7555B129B488005130D4EDA426733D552D10B36D8F",
+      "name": "MUSE",
+      "display": "muse",
+      "symbol": "MUSE",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "cw20:juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/MUSE.png",
```

Comment:
please point these img urls instead to the chain registry. E.g.,
"https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/muse.svg"

Reply:
Need to PR to upload image at this path?

Reply:
updated

Reply:
would be best to have these imgs at the Chain Registry, yes. I only see muse.png there...

Reply:
I must create new PR for svg?
can we link for svg this site?

Reply:
I deleted svg.